django/db/backends/base/creation.py

```diff
@@ -268,6 +272,26 @@ def destroy_test_db(self, old_database_name=None, verbosity=1, keepdb=False, num
             settings.DATABASES[self.connection.alias]["NAME"] = old_database_name
             self.connection.settings_dict["NAME"] = old_database_name
 
+    def _restore_initial_data_migrations(self):
+        # Load initial data migrations, if any.
+        if hasattr(connections[self.connection.alias], "_test_serialized_contents"):
+            self._flush_db()
```

Comment:
That was the only way I thought about to mock it properly in tests, as the `call_command` is imported locally...